includes/admin/amp-backend-actions.php

```diff
@@ -0,0 +1,46 @@
+<?php
+// Callbacks for adding AMP-related things to the admin.
+
+/** AMP_Template_Customizer class */
+require_once( AMP__DIR__ . '/includes/admin/class-amp-customizer.php' );
+
+/**
+ * Instantiates the AMP template editor for the Customizer.
+ *
+ * @param WP_Customize_Manager $wp_customize Customizer instance.
+ */
+function init_amp_template_customizer( $wp_customize ) {
+	AMP_Template_Customizer::init( $wp_customize );
+}
+add_action( 'customize_register', 'init_amp_template_customizer', 500 );
+
+/**
+ * Registers a submenu page to access the AMP template editor panel in the Customizer.
+ */
+function amp_customizer_editor_link() {
+	$post_id = get_posts( array(
+		'post_status'      => 'publish',
+		'post_type'        => 'post',
+		'orderby'          => 'rand',
```

Comment:
This is covered in 0af694acf where I've removed `rand` and made the post type filterable.